packages/webapp/src/components/Animals/Inventory/index.tsx

```diff
@@ -59,6 +59,7 @@ const PureAnimalInventory = ({
   clearFilters,
   isLoading,
   containerHeight,
+  isWorker,
```

Comment:
I'm sorry I should've asked this earlier, but would you mind changing `isWorker` to `isAdmin`? It would help make it clearer and more consistent throughout the app!